VRCFaceTracking/Params/Expressions/UnifiedExpressionsMerger.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VRCFaceTracking.Params
+{
+    public static class UnifiedExpressionsMerger
+    {
+        public static readonly IParameter[] UnifiedCombinedShapes =
+        {    
+            // Unified Eye Definitions
+            
```

Comment:
May need to name v2 to something else that more useful to understand in the parameter drop down box. This should be permanent goal to have it in a menu for organization. So  i.e. FT/LeftEyeX

Reply:
v2/... is to indicate it is using the new standard, I can make it so that you would be able to attach any extensions you would want (in the case of organization).